pkgs/development/python-modules/black/default.nix

```diff
@@ -20,13 +21,13 @@
 
 buildPythonPackage rec {
   pname = "black";
-  version = "21.7b0";
+  version = "21.8b0";
 
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "06d27adq6v6p8wspi0wwqz2pnq34p5jhnqvijbin54yyj5j3qdy8";
+    sha256 = "sha256-VwYI0oqjrxeSuYxKM326xjZ4d7R7EriKtCCVz8GmJ8I=";
```

Comment:
`hash = ... ` is less redundant and more common

Reply:
We should stick to `sha256` for now. As long as there is no decision made about this (https://github.com/NixOS/nixpkgs/pull/112388) and support present in the tools for auto-conversion (e.g., `nix-update`).

Reply:
Ok. Thanks for the explanation and for introducing me to `nix-update` :smile: